sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.0.1 (Unreleased)
 
+- The `text` property of the `SentenceSentiment` interface is no longer optional because the service always returns it.
```

Comment:
It would be good to mention that this interface is used only in the output types so that it does not alarm the users.

Reply:
Done.